site/docs/uk/hosting/vps.md

```diff
@@ -41,7 +141,7 @@ systemd --- це потужний менеджер служб, який пере
    # /home/user/.nvm/versions/node/v16.9.1/bin/node /home/user/bot1/index.js
```

Comment:
```suggestion
   # Приклад для Node.js:
   # /home/user/.nvm/versions/node/v16.9.1/bin/node /home/user/bot1/index.js
```

Reply:
What? There seems to be no problem here
![Screenshot from 2024-02-24 17-48-36](https://github.com/grammyjs/website/assets/75927667/92676cfe-8534-4c74-b9bf-d8cc52c44c0c)

Reply:
Oops, my bad.

Reply:
My bad.